manifests/r/RustemMussabekov/Raindrop/5.6.2/RustemMussabekov.Raindrop.locale.en-US.yaml

```diff
@@ -0,0 +1,38 @@
+# Created with YamlCreate.ps1 v2.2.7 $debug=QUSU.CRLF.7-3-4.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: RustemMussabekov.Raindrop
+PackageVersion: 5.6.2
+PackageLocale: en-US
+Publisher: Rustem Mussabekov
+PublisherUrl: https://raindrop.io
+PublisherSupportUrl: https://github.com/raindropio/desktop/issues
+PrivacyUrl: https://help.raindrop.io/privacy
+Author: Rustem Mussabekov
+PackageName: Raindrop.io
+PackageUrl: https://raindrop.io/download
+License: MIT
```

Comment:
I can't find a reference to MIT license at the LicenseUrl. If we can't find concrete info about the package license, we should  use `License: Proprietary` instead

Reply:
I've gone though github repos of the app and haven't found anything about license. So I also think that license should be changed to proprietary. I really don't understand where the person who submitted the package for the first time found info that it's licensed under MIT License.

Reply:
I looked even into history of repo using wayback machine and didn't find any mentions about MIT License.

Reply:
@krystofkrticka You'll have to mark this conversation as resolved to unblock auto-merge